<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8" />

    <title>GTC - Curation documentation</title>

    <meta name="description" content="EMBL-EBI Genome Targeting Catalogue" /><!-- Describe what this page is about -->
    <meta name="keywords" content="crispr, genome targeting, genome editing" />
    <!-- 3 to 10 keywords about the content of this page (not the whole project) -->
    <meta name="author" content="EMBL-EBI" /><!-- Your [project-name] here -->
    <meta name="HandheldFriendly" content="true" />
    <meta name="MobileOptimized" content="width" />
    <meta name="viewport" content="width=device-width,initial-scale=1" />
    <meta name="theme-color" content="#70BDBD" /> <!-- Android Chrome mobile browser tab color -->
    <!-- Get suggested SEO and social metatags at:
             https://www.ebi.ac.uk/style-lab/websites/patterns/meta-copy.html -->

    <!-- Add information on the life cycle of this page -->
    <meta name="ebi:owner" content="John Doe" /> <!-- Who should be contacted about changes -->
    <meta name="ebi:review-cycle" content="0" /> <!-- In days, how often should the content be reviewed -->
    <meta name="ebi:last-review" content="2020-08-01" /> <!-- The last time the content was reviewed -->
    <meta name="ebi:expiry" content="2025-01-01" /> <!-- When this content is no longer relevant -->

    <!-- If you link to any other sites frequently, consider optimising performance with a DNS prefetch -->
    <link rel="dns-prefetch" href="//www.ebi.ac.uk" />

    <!-- If you have custom icon, replace these as appropriate.
             You can generate them at realfavicongenerator.net -->
    <link rel="icon" type="image/x-icon"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/favicon.ico" />
    <link rel="icon" type="image/png"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/favicon-32x32.png" />
    <link rel="icon" type="image/png" sizes="192x192"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/android-chrome-192x192.png" />
    <!-- Android (192px) -->
    <link rel="apple-touch-icon-precomposed" sizes="114x114"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/apple-icon-114x114.png" />
    <!-- For iPhone 4 Retina display (114px) -->
    <link rel="apple-touch-icon-precomposed" sizes="72x72"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/apple-icon-72x72.png" />
    <!-- For iPad (72px) -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/apple-icon-144x144.png" />
    <!-- For iPad retinat (144px) -->
    <link rel="apple-touch-icon-precomposed"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/apple-icon-57x57.png" />
    <!-- For iPhone (57px) -->
    <link rel="mask-icon"
        href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/safari-pinned-tab.svg"
        color="#ffffff" /> <!-- Safari icon for pinned tab -->
    <meta name="msapplication-TileColor" content="#2b5797" /> <!-- MS Icons -->
    <meta name="msapplication-TileImage"
        content="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/images/logos/EMBL-EBI/favicons/mstile-144x144.png" />

    <!-- CSS: implied media=all -->
    <!-- CSS concatenated and minified via ant build script-->

    <link rel="stylesheet" href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/css/ebi-global.css"
        type="text/css" media="all" />
    <link rel="stylesheet" href="https://dev.ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.2/fonts.css"
        type="text/css" media="all" />

    <!-- Use this CSS file for any custom styling -->
    <!-- <link rel="stylesheet" href="/gtc/static/css/custom.css" type="text/css" media="all" /> -->

    <!-- If you have a custom header image or colour -->
    <!--
        <meta name="ebi:masthead-color" content="#000" />
        <meta name="ebi:masthead-image" content="//www.ebi.ac.uk/web_guidelines/EBI-Framework/images/backgrounds/embl-ebi-background.jpg" />
        -->

    <!-- you can replace this with theme-[projectname].css. See http://www.ebi.ac.uk/web/style/colour for details of how to do this -->
    <!-- also inform ES so we can host your colour palette file -->
    <link rel="stylesheet" href="https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/css/theme-embl-petrol.css"
        type="text/css" media="all" />
    <!-- end CSS-->
</head>

<body class="level2">
    <div id="skip-to">
        <a href="#content">Skip to main content</a>
    </div>
    <header id="masthead-black-bar" class="clearfix masthead-black-bar">
    </header>


    <div id="content">
        <div data-sticky-container>
            <header id="masthead" class="masthead" data-sticky data-sticky-on="large" data-top-anchor="content:top"
                data-btm-anchor="content:bottom">
                <div class="masthead-inner row">
                    <!-- local-title -->
                    <div class="columns medium-12" id="local-title">
                        <a href="/gtc/">
                            <img class="logo" src="/gtc/img/GTC_master_logos_inverted.png" width="130" height="60" alt="GTC Logo"
                                style="float: left;">
                        </a>
                        <h1><a href="/gtc/">&nbsp; Genome Targeting Catalogue</a></h1>
                    </div>
                    <!-- /local-title -->
                    <!-- local-nav -->
                    <nav>
                        <ul id="local-nav" class="dropdown menu float-left" data-description="navigational">
                            <li><a href="/gtc/">Overview</a></li>
                            <li><a href="/gtc/publications">Publications</a></li>
                            <li><a href="/gtc/curation">Curation Documentation</a></li>
                        </ul>
                    </nav>
                    <!-- /local-nav -->
                </div>
            </header>
        </div>
        <!-- Suggested layout containers -->
        <section id="main-content-area" class="row" role="main">
            <!-- Your menu structure should make a breadcrumb redundant, but if a breadcrump is needed uncomment the below -->
            <nav aria-label="You are here:" role="navigation">
                <ul class="breadcrumbs columns">
                    <li><a href="/gtc/">Genome Targeting Catalogue</a></li>
                    <li>
                        <span class="show-for-sr">Current: </span> Curation documentation
                    </li>
                </ul>
            </nav>

            <div class="columns">
                <section>

                    <div class="columns">
                        <section>
                            <h2>Curation Criteria and Data Submission Guide</h2>
                            <p> This page describes the criteria for publications that are curated into the database.
                                If any of the information in this document is unclear, or if you have questions about
                                why a
                                particular paper was not included in the database, please email
                                <a href="mailto:gtc@ebi.ac.uk">gtc@ebi.ac.uk</a>. </p>
                        </section>

                        <section>
                            <h3 id="paper-curation">Paper curation</h3>
                            <p>The flow chart below is used to assess whether a paper contains
                                sufficient information to be included in the database.</p>
                            <img src="/gtc/img/accept_paper_flowchart.png" alt="accept_paper_flowchart.png" style="width: 60%">
                            <br/><br/>
                            <p> If the paper fulfils the criteria listed above then it will potentially be suitable for
                                curation.
                                Supplementary files are also taken into consideration. </p>
                            <p> Additional requirements for curation are given below, these criteria aim to standardise
                                incoming
                                papers and to aid in programmatic accession: </p>
                            <ul>
                                <li>The species in the paper must have a sequenced genome in
                                    <a href="http://www.ensembl.org">Ensembl</a> or
                                    <a href="http://ensemblgenomes.org">Ensembl Genomes</a>.
                                </li>
                                <li>The gene targets must be mapped to the sequenced genome.</li>
                                <li>Cell lines must be described in one of the ontologies available in the
                                    <a href="https://www.ebi.ac.uk/ols">Ontology Lookup Service</a>.
                                    This includes the potential for compound terms for primary cell lines..
                                </li>
                            </ul>
                            <p> If a paper is found to fulfil these criteria then the required information can be
                                entered into
                                an excel file to be programmatically entered into the database. </p>
                            <p> If at any point it becomes clear that a paper does not meet the criteria listed here,
                                or lacks key information required to make the data comparable and repeatable,
                                it will be deemed low priority. Low priority papers are not entered into the database,
                                independent of the level of curation already carried out. If further information becomes
                                available, for example through author submission, the paper can be included in the
                                database. </p>

                            <h3 id="data-submission-guide">Data Submission Guide</h3>
                            <p>Data can be submitted for the database in the form of an Excel spreadsheet with metadata
                                related
                                to the publication and the experiments contained within it.</p>
                            <p>The first step is to download the curation form and change the file name to include the
                                first author of the paper, the PubMed ID of the paper and the year it was published in.
                                These should be separated by an underscore for ease of programmatic analysis.
                                The PubMed ID (PMID) can be found by searching for the paper in
                                the <a href="http://europepmc.org/">Europe PMC</a> database.</p>
                            <p> For example: </p>
                            <p><i>Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J. A., & Charpentier, E.
                                    (2012).
                                    A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity.
                                    Science, 337,
                                    816–822.</i>
                            </p>
                            <p>Would be given the file name:</p>
                            <p><i>Jinek_22745249_2012.xlsx</i></p>

                            <p> Please do not include the names of multiple first authors or use any accented letters or
                                apostrophes.
                                Please write compound surnames in Pascal case, for example van der Waals would be
                                written as
                                VanDerWaals, and Barré-Sinoussi would be written as BarreSinoussi </p>
                            <p> The aim of the curation process is to complete as many of the following fields as
                                possible
                                using the information found in the main text and supplementary information of the
                                publication.
                                Fields in which a response is required are indicated with an asterisk and also
                                highlighted
                                in red in the spreadsheet. </p>
                            <p> Where the letter D is shown in the second column this indicates that there is a drop-box
                                containing options. Drop-down lists have been populated using data from papers that
                                have been curated; this requires the regular addition of new options.
                                Where an answer is required that has not been provided by the drop-down box
                                please email us at <a href="mailto:gtc@ebi.ac.uk">gtc@ebi.ac.uk</a> and the option will
                                be added. </p>

                            <h4 id="publication">1. Publication</h4>
                            <p>This section of the form records information about the paper, including the PubMed ID
                                which makes
                                it easy to retrieve the paper and capture any additional information required. </p>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td class="required-field">First author</td>
                                        <td></td>
                                        <td>The full name of the first author as written on the published paper.
                                            Only one first author is allowed.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Title</td>
                                        <td></td>
                                        <td>Full title of the paper as published.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">PubMed ID</td>
                                        <td></td>
                                        <td>PMID of the paper.</td>
                                    </tr>
                                    <tr>
                                        <td>CRISPR related method papers cited (PMID)</td>
                                        <td></td>
                                        <td>This field is for use when the methods section of the paper references other
                                            papers
                                            for additional details of genome targeting experiments. Multiple PMIDs
                                            should be separated by a comma.</td>
                                    </tr>
                                    <tr>
                                        <td>Other technique(s)</td>
                                        <td>D</td>
                                        <td>Recording any additional techniques used to target the same genomic region
                                            or protein.
                                            This could include morpholinos, siRNA or another genome targeting technique
                                            such as Zinc Finger Nucleases.</td>
                                    </tr>
                                    <tr>
                                        <td>Data type</td>
                                        <td>D</td>
                                        <td>
                                            <p>In the event that data has been saved in a repository, this field and the
                                                two following should be used to record the location and type of data.
                                            </p>
                                            <p>This field describes the type of data that has been placed in a database,
                                                for example sequencing data of guide strand RNA sequences.
                                                Give all data types for each of the accession numbers.</p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Archive</td>
                                        <td>D</td>
                                        <td>The name of the database into which the data was entered.
                                            Give a database for each of the accession numbers.</td>
                                    </tr>
                                    <tr>
                                        <td>Accession Number</td>
                                        <td>D</td>
                                        <td>In separate cells give the accession numbers of the data. Each data type
                                            should have
                                            an accession number, numbers can be repeated if necessary. This field
                                            remains
                                            blank if the raw data has been saved on the journal website.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Are off-target effects studied?</td>
                                        <td>D</td>
                                        <td>The extent to which off-target effects are studied can include sequencing
                                            experiments
                                            around predicted off-target site or simply discussing the steps taken to
                                            prevent
                                            off-target binding. If there is no discussion or experimentation regarding
                                            off-target effects then this should also be indicated.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Genome-wide/arrayed study included?</td>
                                        <td></td>
                                        <td>Whether there is a genome-wide CRISPR study in the publication can be
                                            indicated in this field.
                                            This section provides programmatic direction that there will be relevant
                                            information
                                            in Section 4.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Targeted study included?</td>
                                        <td>D</td>
                                        <td>Whether there is a CRISPR study with selected targets in the publication can
                                            be indicated
                                            in this field. This section provides programmatic direction that there will
                                            be relevant
                                            information in Section 3.</td>
                                    </tr>
                                </tbody>
                            </table>

                            <h4 id="study-information">2. Study information</h4>
                            <p>This section allows the alignment of information about different studies carried out
                                within
                                the same paper. It is useful to think about a study in terms of an imaginary test tube;
                                if you find that you would not add a reagent or cell line to the same test tube then
                                it represents a separate study. The different species, cell types and experimental
                                variables can all be recorded together in this section. </p>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td class="required-field">Internal key</td>
                                        <td></td>
                                        <td>
                                            <p> The internal key indicates a number given to each study as an internal
                                                reference
                                                in order to connect data describing the same studies between different
                                                sections
                                                of the accession spreadsheet. Each study described in the paper using a
                                                different
                                                set of variables should be described in this section and assigned a
                                                number sequentially.
                                                There is no weight attached to the numbering; it can be carried out in
                                                any order that
                                                make sense to the person filling in the spreadsheet. </p>
                                            <p> The internal reference should be set in Section 2.
                                                Study Information and then applied to the further description of
                                                the studies in Section 3. and Section 4. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Assay type</td>
                                        <td>D</td>
                                        <td>The desired method of action of the CRISPR associated protein and the
                                            experimental
                                            output. If the study aims to find which guide strand RNA sequences are
                                            present or absent
                                            in the population following enzyme mediated NHEJ, then the study would
                                            be “Knockout by NHEJ – proliferation.” Tiling of fluorescently labelled
                                            protein
                                            can be used for imaging, or tags can be attached to the CRISPR associated
                                            protein
                                            to aid transcriptional repression or activation.</td>
                                    </tr>
                                    <tr>
                                        <td>HDR Donor</td>
                                        <td>D</td>
                                        <td>Where a study has been designed to carry out homology directed repair (HDR)
                                            of the strand
                                            breaks caused by the nuclease, indicate the form of the template DNA added
                                            for repair,
                                            for example a single-stranded donor oligonucleotide (ssODN).</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Enzyme variant</td>
                                        <td>D</td>
                                        <td>The enzyme used (e.g. Cas9, Cpf1) and any additional information given. For
                                            example,
                                            the species from which the sequence was taken, whether the sequence has been
                                            humanised
                                            (i.e. codon optimised for expression in human cells), whether there are any
                                            additional
                                            protein domains fused to the CRISPR associated protein (e.g. KRAB for
                                            transcriptional
                                            repression, fluorescent compounds for FACS analysis or for imaging).</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Inducible?</td>
                                        <td>D</td>
                                        <td>Indicate whether the study has expressed the CRISPR-related protein or sgRNA
                                            in an
                                            inducible manner.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Multiplexed or multitarget delivery?</td>
                                        <td>D</td>
                                        <td>
                                            <p> Indicate whether the study has aimed to introduce more than one single
                                                guide
                                                RNA to the cells or tissue, targeting the same gene or genomic region
                                                (multiplex).
                                                It can also be used to indicate where the guide strand or strands were
                                                designed
                                                to target multiple genes concurrently (multitarget). </p>
                                            <p> While it is possible that multiple guide strands may enter cells in any
                                                pooled study,
                                                this field is used to indicate when the delivery of multiple guide
                                                strands was part
                                                of the experimental aim. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Organism ID</td>
                                        <td></td>
                                        <td>A list of species is provided, giving a reference number which links to a
                                            sequenced
                                            genome in Ensembl or Ensembl Genomes. The reference number, e.g. 170 for
                                            Homo sapiens,
                                            should be recorded in this field.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Species</td>
                                        <td>D</td>
                                        <td>The scientific name of the species used in the study or studies.</td>
                                    </tr>
                                    <tr>
                                        <td>Strain</td>
                                        <td></td>
                                        <td>The strain of the species can be recorded here. Strains not found in Ensembl
                                            or the Ontology Lookup Service (OLS) can be recorded here.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Biosample IDs</td>
                                        <td></td>
                                        <td>The IDs of the cell types and strains used in the study should be recorded
                                            in this field,
                                            for example the EFO ID for HeLa cells is EFO_0001185. IDs should be taken
                                            from the Ontology
                                            Lookup Service (OLS) and separated by commas when multiple entries are
                                            necessary.</td>
                                    </tr>
                                    <tr>
                                        <td>Biosample modifications</td>
                                        <td></td>
                                        <td>This field is used to record any additions or changes made to cell lines
                                            before their use
                                            in a targeting experiment. This could include the introduction of a plasmid
                                            containing
                                            a fluorescent reporter, or an antibiotic selection marker.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Total guide strand RNAs</td>
                                        <td></td>
                                        <td>The total number of guide strands used in conjunction with the cell line/
                                            strain of organism.</td>
                                    </tr>
                                    <tr>
                                        <td>Genes targeted</td>
                                        <td></td>
                                        <td>The number of genes for which guide strand RNA sequences were designed.</td>
                                    </tr>
                                    <tr>
                                        <td>Positive controls</td>
                                        <td></td>
                                        <td>The number of guide strand RNA sequences that were designed to act as
                                            positive controls,
                                            i.e. the targets of the sequences had a known phenotype and this could be
                                            measured
                                            to assess the activity of the experimental output.</td>
                                    </tr>
                                    <tr>
                                        <td>Negative controls</td>
                                        <td></td>
                                        <td>The number of guide strand RNA sequences that were designed to act as
                                            negative controls,
                                            i.e. the sequences had no known targets in the organism and their output
                                            could be
                                            used to define the background to the experimental output.</td>
                                    </tr>
                                    <tr>
                                        <td>Biological replicates</td>
                                        <td></td>
                                        <td>The number of biological replicates carried out in the study, for example
                                            non-sister
                                            clones or different animals.</td>
                                    </tr>
                                    <tr>
                                        <td>Technical replicates</td>
                                        <td></td>
                                        <td>The number of times the study was carried out using the same material, for
                                            example the
                                            same clone or tissue. Where no information is given to the contrary, it will
                                            be
                                            assumed that replicates are technical replicates.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Enzyme type</td>
                                        <td>D</td>
                                        <td>The form in which the enzyme is delivered to the cells or organisms, for
                                            example as
                                            an insert in a plasmid or as a ribonucleoprotein (RNP) complex.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Enzyme delivery</td>
                                        <td>D</td>
                                        <td>The method by which the protein is delivered to the cells or organisms in
                                            the study.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">sgRNA type</td>
                                        <td></td>
                                        <td>The form in which the sgRNA is delivered to the cells or organisms, for
                                            example as
                                            an insert in a plasmid or as a ribonucleoprotein (RNP) complex.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">sgRNA delivery</td>
                                        <td>D</td>
                                        <td>The method by which the guide strand RNA sequences are delivered to the
                                            cells or
                                            organisms in the study.</td>
                                    </tr>
                                    <tr>
                                        <td>Drug/condition treated</td>
                                        <td>D</td>
                                        <td>The growth conditions or drugs used as a selection pressure for the cells to
                                            identify
                                            the phenotype of the targeted cells, for example low nutrient media or high
                                            temperature.</td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">sgRNA design tool</td>
                                        <td>D</td>
                                        <td>The tool used to design and optimise the guide strand RNA sequences. When
                                            in-house design
                                            was carried out or the library was purchased this should also be indicated.
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h4 id="guide-strand-sequences-target">3. Guide strand sequences (Targeted)</h4>
                            <p>Where a small number of genes have been targeted it should be possible to record the gene
                                targets,
                                genomic coordinates and guide strand RNA sequences, as well as the efficiencies of the
                                strands
                                and the metric used to assess the efficiency. If a genome-wide screen has been used in
                                the study,
                                this sheet can either be left blank or can be used to record the details of preliminary
                                or
                                follow-up studies carried out on a smaller number of targets.</p>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td class="required-field">Internal key</td>
                                        <td></td>
                                        <td>
                                            <p> The internal reference number assigned to the study in Section 2. should
                                                be repeated here. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Ensembl ID</td>
                                        <td></td>
                                        <td>
                                            <p> The Ensembl symbol for the gene or genomic region to which the guide
                                                strand RNA directs
                                                the CRISPR associated protein. Gene targets must be verified using
                                                Ensembl or Ensembl Genomes. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Reported gene name</td>
                                        <td></td>
                                        <td>
                                            <p> If the gene is given a different name in the paper this can be recorded
                                                here,
                                                otherwise this field should be left blank. Only gene synonyms from
                                                Ensembl can be accepted.
                                            </p>
                                            <p>If a reported name isn’t in Ensembl but can be found via RefSeq, the
                                                Refseq reference
                                                should be given as well as the reported gene name, separated by a comma.
                                            </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Transgene?</td>
                                        <td>D</td>
                                        <td>
                                            <p> This field indicates whether the gene targeted is endogenous or whether
                                                it has
                                                been introduced to the biological system. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Gene Symbol</td>
                                        <td></td>
                                        <td>
                                            <p> When a gene is a transgene, this is the gene symbol. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Gene Description</td>
                                        <td></td>
                                        <td>
                                            <p> When a gene is a transgene, this is the description. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Synonyms</td>
                                        <td></td>
                                        <td>
                                            <p> When a gene is a transgene, these are the synonyms. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Genomic coordinates</td>
                                        <td></td>
                                        <td>
                                            <p> The genomic coordinates of the guide strand RNA sequence. If this is not
                                                provided
                                                by the authors, then this field is left blank. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Genome assembly</td>
                                        <td>D</td>
                                        <td>
                                            <p> The genome assembly that the guide strand RNA sequence has been aligned
                                                to. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Guide strand RNA sequence</td>
                                        <td></td>
                                        <td>
                                            <p> The 20 base targeting sequence is sufficient for this field. It is also
                                                possible to put
                                                the full sgRNA or crRNA and tracrRNA sequences if these are given. </p>
                                            <p> This field should always be completed, except when a commercial sgRNA
                                                has been used.
                                                In the event that a commercially sourced sgRNA has been used and no
                                                sequence is given,
                                                then this field might be left blank provided the fields sgRNA Source and
                                                Source ID are completed. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>sgRNA label</td>
                                        <td></td>
                                        <td>
                                            <p> If identifiers are given to the guide strand RNA sequences in the paper,
                                                then these are recorded in this field. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Guide strand efficiency</td>
                                        <td></td>
                                        <td>
                                            <p> Any measure of how efficient the guide strand RNA sequence was. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Metric</td>
                                        <td>D</td>
                                        <td>
                                            <p> The method by which the efficiency of the guide strand RNA sequence was
                                                measured. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">sgRNA source</td>
                                        <td>D</td>
                                        <td>
                                            <p> Where a commercially sourced sgRNA has been used, this field should be
                                                used to indicate
                                                the sources from which it was purchased. </p>
                                            <p> This field and the Source ID field are only required when a commercial
                                                sgRNA has been
                                                used and no sequence is given. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Source ID</td>
                                        <td></td>
                                        <td>
                                            <p> An identifier, for example the catalogue number, of the commercial sgRNA
                                                used in this study.
                                                The sgRNA sequence of commercial sgRNAs is not usually given, so as much
                                                effort as possible
                                                should be taken to ensure this is a stable identifier. </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h4 id="guide-strand-sequences-genome-arrayed">4. Guide strand sequences (Genome-wide or
                                arrayed)</h4>
                            <p>Where a genome-wide screen has been used, either in pooled format or in an array, the
                                information about
                                the guide strand library and some broad experimental details can be recorded. If only a
                                small number of
                                genes were targeted, then this sheet can be left blank.</p>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td class="required-field">Internal key</td>
                                        <td></td>
                                        <td>
                                            <p> The internal reference number assigned to the study in Section 2. should
                                                be repeated here. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">sgRNA library name</td>
                                        <td>D</td>
                                        <td>
                                            <p> The commercial or lab-generated library used in the screen study. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Sublibrary</td>
                                        <td>D</td>
                                        <td>
                                            <p> The commercial or lab-generated sublibrary used in the screen study.
                                            </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>sgRNA/ gene</td>
                                        <td></td>
                                        <td>
                                            <p> The number of guide strands targeted to each gene in the library or
                                                sublibrary used
                                                in the screen study. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Library source</td>
                                        <td>D</td>
                                        <td>
                                            <p> The company from which the library was purchased, or the lab from which
                                                it was received. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Genome assembly</td>
                                        <td></td>
                                        <td>
                                            <p> The genome assembly to which the sequences were mapped. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Analysis tool</td>
                                        <td>D</td>
                                        <td>
                                            <p> The computational method used for analysing the data from the screen
                                                study. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Analysis tool version</td>
                                        <td></td>
                                        <td>
                                            <p> The version of the computational analysis tool used. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td class="required-field">Efficiency metric</td>
                                        <td>D</td>
                                        <td>
                                            <p> The statistical test used by the authors of the study for assessing the
                                                outcome of the study. </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h4 id="experimental-detail">5. Experimental detail (Genome-wide)</h4>
                            <p>Experimental detail is not a searchable section of the database and is only provided for
                                papers
                                describing genome-wide screens. This data can be provided by researchers who are
                                curating their
                                own publication and wish to share more of the experimental information in a standardised
                                format.</p>
                            <h5 id="general-information">5.1 General Information</h5>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td>Plasmids used</td>
                                        <td>
                                            <p> The names of the plasmids that were used for screening, e.g. those used
                                                to deliver
                                                the CRISPR-related protein sequence. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Production of plasmid library</td>
                                        <td>
                                            <p> The protocol used to produce the plasmid library. </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h5 id="production-of-viral-libraries">5.2 Production of viral libraries</h5>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td>Packaging cell line</td>
                                        <td>
                                            <p> The name of the cell line that was used to support the packaging of the
                                                RNA strands
                                                into the virus. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Packaging plasmids and viral protein</td>
                                        <td>
                                            <p> The names of the packaging plasmids and the viral protein used to
                                                package the RNA. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Growth Conditions/Cell culture conditions to generate virus library</td>
                                        <td>
                                            <p> Copy an overview of the protocols used for cell line culturing. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Percentage of cell confluence at day of transfection</td>
                                        <td>
                                            <p> Give the percentage cell confluency when the cells were transduced. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Transfection protocol</td>
                                        <td>
                                            <p> An overview of the transfection protocol. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Obtaining viral supernatant protocol</td>
                                        <td>
                                            <p> Information on how the viral supernatant was obtained and processed.
                                            </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h5 id="production-of-stable-library">5.3 Production of stable library cell line and Cas9
                                information</h5>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td>Growth Conditions/Cell culture conditions</td>
                                        <td>
                                            <p> The name of the cell line that was used to support the packaging of the
                                                RNA strands
                                                into the virus. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Transduction protocol</td>
                                        <td>
                                            <p> Provide the transduction protocol, including the multiplicity of
                                                infection (MOI)
                                                used for infection if it is given. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Antibiotic selection</td>
                                        <td>
                                            <p> Provide the name of the selection marker and the concentration used, if
                                                given. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Time period for selection</td>
                                        <td>
                                            <p> The length(s) of time between the introduction of CRISPR protein and/or
                                                guide strand RNA,
                                                allowing the production of the active complex, and assessment of the
                                                phenotype of the cells. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Cultivation protocol during selection</td>
                                        <td>
                                            <p> The protocol used for cell culture during the selection process. </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>

                            <h5 id="library-and-sequencing-preparation">5.4 Library and Sequencing Preparation</h5>
                            <table class="table table-striped">
                                <tbody>
                                    <tr>
                                        <td>Number of cells harvested per sample</td>
                                        <td>
                                            <p> The number of approximate number of cells harvested for gDNA extraction
                                                per sample. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>PCR Protocol</td>
                                        <td>
                                            <p> A detailed overview of the PCR protocol including enzyme names, cycling
                                                times etc. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Amount of input DNA per PCR</td>
                                        <td>
                                            <p> The amount of gDNA used as input for each PCR reaction. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Primer(s) for PCR</td>
                                        <td>
                                            <p> The name(s) and sequence(s) of the primer(s) used for sequencing. </p>
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Sequencing</td>
                                        <td>
                                            <p> A description of the sequencing parameters. </p>
                                        </td>
                                    </tr>
                                </tbody>
                            </table>
                        </section>
                    </div>

                </section>
            </div>
        </section>
        <!-- Optional local footer (insert citation / project-specific copyright / etc here -->
        <footer>
            <br>
            <div class="row">
                <div align="center" id="contact-footer">
                    <p>
                        <a class="icon icon-socialmedia" data-icon="T" href="https://twitter.com/gt_catalogue">
                            @gt_catalogue</a>
                    </p>
                    <p>
                        <a class="icon icon-generic" data-icon="E" href="mailto:gtc@ebi.ac.uk"> gtc@ebi.ac.uk</a>
                    </p>
                </div>
        </footer>
    </div>
    <!-- End suggested layout containers / #content -->

    <footer>
        <div id="global-footer" class="global-footer">
            <nav id="global-nav-expanded" class="global-nav-expanded row">
                <!-- Footer will be automatically inserted by footer.js -->
            </nav>
            <section id="ebi-footer-meta" class="ebi-footer-meta row">
                <!-- Footer meta will be automatically inserted by footer.js -->
            </section>
        </div>
    </footer>

    <!-- JavaScript -->
    <!-- Grab Google CDN's jQuery, with a protocol relative URL; fall back to local if offline -->
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
    <!--
        <script>window.jQuery || document.write('<script src="../js/libs/jquery-1.10.2.min.js"><\/script>')</script>
        -->

    <script defer="defer" src="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/js/script.js"></script>

    <!-- The Foundation theme JavaScript -->
    <script
        src="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/libraries/foundation-6/js/foundation.js"></script>
    <script src="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/js/foundationExtendEBI.js"></script>
    <script type="text/JavaScript">$(document).foundation();</script>
    <script type="text/JavaScript">$(document).foundationExtendEBI();</script>

    <!-- Google Analytics details... -->
    <!-- Change UA-XXXXX-X to be your site's ID -->
    <script
        type="text/javascript">!function (e, a, n, t, i, o, c) {
                e.GoogleAnalyticsObject = i, e[i] = e[i] || function () {
                    (e[i].q = e[i].q || []).push(arguments)
                }, e[i].l = 1 * new Date, o = a.createElement(n), c = a.getElementsByTagName(n)[0], o.async = 1, o.src = "//www.google-analytics.com/analytics.js", c.parentNode.insertBefore(o, c)
            }(window, document, "script", 0, "ga"), ga("create", "UA-629242-1", { cookieDomain: "auto" }), ga("require", "linkid", "linkid.js"), ga("set", "anonymizeIp", !0), ga("send", "pageview");</script>
</body>

</html>